core/aws.py

```diff
@@ -0,0 +1,36 @@
+from io import FileIO
+
+from boto3 import client
+from botocore.exceptions import ClientError
+from flask import current_app
+
+from config import Config
+
+if hasattr(Config, "AWS_ACCESS_KEY_ID") and hasattr(Config, "AWS_SECRET_ACCESS_KEY"):
+    _S3_CLIENT = client(
+        "s3",
+        aws_access_key_id=Config.AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=Config.AWS_SECRET_ACCESS_KEY,
+        region_name=Config.AWS_REGION,
+        endpoint_url=Config.AWS_ENDPOINT_OVERRIDE,
+    )
+else:
+    # boto gets access keys from the environment directly in AWS
+    _S3_CLIENT = client(
+        "s3",
+        region_name=Config.AWS_REGION,
+        endpoint_url=Config.AWS_ENDPOINT_OVERRIDE,
+    )
+
+
+def upload_file(file: FileIO, bucket: str, object_name: str):
```

Comment:
Can `BUCKET_NAME` be an env var? If there are multiple buckets they can be separate env vars.

Reply:
It is already an envar currently. We have 2 buckets configured in AWS at the moment linked to the data-store